evil-vars.el

```diff
@@ -1280,6 +1280,11 @@ having higher priority.")
 (defvar evil-command-properties nil
   "Specifications made by `evil-define-command'.")
 
+(defvar evil-change-commands '(evil-change)
+	"Commands that wrap or replace `evil-change'.
```

Comment:
One more nitpick, indentation is off here.

Reply:
ah, sorry, Fixing now.